src/msbuild/ConvertToVersionRange.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
```

Comment:
Would it be possible to create a test or two around this so we know if someone or something breaks this logic in the future?

Reply:
Also will there be some documentation for this new target somewhere?

Reply:
> Would it be possible to create a test or two around this so we know if someone or something breaks this logic in the future?

> Also will there be some documentation for this new target somewhere?

It's not possible to include tests in this repo since they would be distributed to all of our repos. Same for documentation.

It's not currently clear where either of those things should be located. For now, we're focused on rolling out the changes, and we'll come back to this later.

Reply:
Ok maybe a separate repo and package would make sense sometime later.

Reply:
> Ok maybe a separate repo and package would make sense sometime later.

This code can't ship as a package. It needs to influence package restore, so has to be there before restore starts.

Reply:
Is there a Test repo that gets RepoStandard updates that could validate changes from RepoStandards?